sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1

```diff
@@ -110,28 +110,25 @@ $dtHostName = az deployment group show -g $ResourceGroup -n $($DigitalTwinName.T
 Write-Host("Set a new client secret for $appId`n")
 $appSecret = az ad app credential reset --id $appId --years 2 --query 'password' --output tsv
 
-$user = $env:UserName
-$fileName = "$user.config.json"
-Write-Host("Writing user config file - $fileName`n")
+$outputfileDir = (Get-Item -Path $PSScriptRoot).Parent.Parent.Parent.Fullname
+$outputFile = Join-Path -Path $outputfileDir -ChildPath "test-resources.json.env"
+$tenantId = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+
+Add-Type -AssemblyName System.Security
+
 $appSecretJsonEscaped = ConvertTo-Json $appSecret
-$config = @"
+$environmentText = @"
 {
-    "DigitalTwinsInstanceHostName": "$dtHostName",
-    "ApplicationId": "$appId",
-    "ClientSecret": $appSecretJsonEscaped,
-    "TestMode":  "Live"
+    "DIGITALTWINS_URL": "$dtHostName",
+    "DIGITALTWINS_CLIENT_ID": "$appId",
+    "DIGITALTWINS_CLIENT_SECRET": $appSecretJsonEscaped,
+    "DIGITALTWINS_TENANT_ID":  "$tenantId"
 }
 "@
 
-$config | Out-File "$PSScriptRoot\..\config\$fileName"
-
-$userSettingsFileSuffix = ".test.assets.config.json"
-$userSettingsFileName = "$user$userSettingsFileSuffix"
-$userTestAssetSettingsFileName = "$PSScriptRoot\..\config\$userSettingsFileName"
-if (-not (Test-Path $userTestAssetSettingsFileName))
-{
-    Write-Host "Creating empty user test assets config file - $userSettingsFileName`n"
-    New-Item -ItemType File -Path $userTestAssetSettingsFileName -Value "{}" | Out-Null
-}
+$bytes = ([System.Text.Encoding]::UTF8).GetBytes($environmentText)
```

Comment:
This will encrypt all the necessary user settings

Reply:
Is there a trick for us to decode the files to view/edit them?